src/diffpy/labpdfproc/labpdfprocapp.py

```diff
@@ -134,11 +141,20 @@ def main():
             metadata=load_metadata(args, filepath),
         )
 
-        absorption_correction = compute_cve(input_pattern, args.mud, args.wavelength)
-        corrected_data = apply_corr(input_pattern, absorption_correction)
+        if args.brute_force:
+            absorption_correction = compute_cve(input_pattern, args.mud, args.wavelength)
+            corrected_data = apply_corr(input_pattern, absorption_correction)
+        else:
+            if args.mud > 6 or args.mud < 0.5:
+                sys.exit(
+                    "mu*D is out of the acceptable range (0.5 to 6) for fast calculation. "
+                    "Please rerun with a value within this range or use -b enable brute-force calculation. "
+                )
+            absorption_correction = fast_compute_cve(input_pattern, args.mud, args.wavelength)
+            corrected_data = apply_fast_corr(input_pattern, absorption_correction)
```

Comment:
is there a need to have a different function for applying the correction?  Surely after computing cve it is the same function to apply it?  Why does this function care how it was computed?

Reply:
No it doesn't care the way it's computed (there are a lot of overlapping between fast and brute-force computation). I think I can remove the duplicated functions in fast_cve.

Reply:
I removed the duplicated functions in fast_cve, so that the module only has one function that takes in a muD and compute the cve for that. I added a simple test on muD=1. I can add more tests on muDs other than 1, but this is probably not necessary?
The process of interpolating the computed cve and applying correction is taken care of by the original functions in functions.py (by passing the brute-force boolean).
In the last commit I moved around some misplaced docstrings in tools.py. I can do this on another PR if it's clearer.